worker/buildbot_worker/runprocess.py

```diff
@@ -814,11 +851,20 @@ def _taskkill(self, pid, force):
                 cmd = "TASKKILL /F /PID {0} /T".format(pid)
             else:
                 cmd = "TASKKILL /PID {0} /T".format(pid)
-
+            if runtime.platformType == 'win32' and self.job_object is not None:
```

Comment:
_taskkill can only be called on win32 platform. Maybe it's worth renaming _taskkill to _win32_taskkill and then removing the above check.

Reply:
Ok, so I renamed the method and remove repetitive check for platformtype.